Object-Containers/src/Homework/objectContainers/MainPerson.java

```diff
@@ -0,0 +1,77 @@
+package Homework.objectContainers;
+
+import java.util.*;
+
+public class MainPerson {
+
+    public static void main(String[] args) {
+
+        // exercise 1
+
+        Person maria = new Person("Maria", 23);
+        Person andrei = new Person("Andrei", 32);
+        Person ion = new Person("Ion", 20);
+
+        PersonAgeComparator p = new PersonAgeComparator();
+
+        Set<Person> setPerson = new TreeSet<>(p);
+        setPerson.add(maria);
+        setPerson.add(andrei);
+        setPerson.add(ion);
+
+        for (Object c : setPerson) {
+            System.out.println(c);
+        }
+
+        // exercise 2
+
+        /**
```

Comment:
Javadoc poti scrie doar pe metode sau pe clase, genul asta de comentariu in interiorul unei metode poate fi inlocuit cu un simplu comentariu single line `//`